[Draft] Use Helm generated instead of cert-manager generated certs for the operator (#1648)
signalfx/splunk-otel-collector-chart: examples/enable-operator-and-auto-instrumentation/otel-demo-nodejs.md

#### 2.1 Deploy the Helm Chart with the Operator enabled

To install the chart with operator in an existing cluster, make sure you have cert-manager installed and available.

> Both the cert-manager and operator are subcharts of this chart and can be enabled with `--set operatorcrds.install=true,operator.enabled=true`.

Reviewer: ```suggestion
The opentelemetry-operator is a subchart of this chart and can be enabled with `--set operatorcrds.install=true,operator.enabled=true`.
```

Not sure why are we mentioning the cert-manager

Author: Yeah my grep missed these in our example demo docs for this specific section. Pushed out a commit to update these docs with your input in mind.
